azimuth/utils/dataset_operations.py

```diff
@@ -62,9 +62,13 @@ def filter_dataset_split(
         and config.columns.text_input in dataset_split.column_names
         and dataset_split.num_rows != 0
     ):
-        by = clean_utterance(filters.utterance)
+        by = filters.utterance
+        by_clean = clean_utterance(by)
```

Comment:
We can get rid of it. And `by_clean` could simply be
```suggestion
        clean = clean_utterance(by)
```